app/src/main/java/com/example/dessertrelease/ui/DessertReleaseScreen.kt

```diff
@@ -108,12 +108,17 @@ private fun DessertReleaseScreen(
 }
 
 @Composable
-fun DessertReleaseLinearLayout(modifier: Modifier = Modifier) {
+fun DessertReleaseLinearLayout(
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(0.dp)
+) {
     LazyColumn(
         modifier = modifier,
         contentPadding = PaddingValues(
-            vertical = dimensionResource(R.dimen.padding_medium),
-            horizontal = dimensionResource(R.dimen.padding_medium)
+            top = contentPadding.calculateTopPadding() + dimensionResource(R.dimen.padding_medium),
+            bottom = contentPadding.calculateBottomPadding() + dimensionResource(R.dimen.padding_medium),
+            start = dimensionResource(R.dimen.padding_medium),
+            end = dimensionResource(R.dimen.padding_medium)
```

Comment:
If you set the required padding to the modifier you can just set scaffold padding to the content padding.
`contentPadding = contentPadding,`